src/components/input/DurationInput.js

```diff
@@ -0,0 +1,404 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { IconChevronUpSmallOutline, IconChevronDownSmallOutline } from '@teamleader/ui-icons';
+import Icon from '../icon';
+import cx from 'classnames';
+import InputBase from './InputBase';
+import { TextBody } from '../typography';
+import Box, { pickBoxProps } from '../box';
+import ValidationText from '../validationText';
+import { durationToSeconds } from '../utils';
+
+import theme from './theme.css';
+
+class SpinnerControls extends PureComponent {
```

Comment:
Not really a fan of having 2 classes in 1 file.

Reply:
I am going to make a separate PR for this change so the `NumericInput` component can also benefit from it.